docs/user/library/jdbc/db2.rst

```diff
@@ -69,6 +69,25 @@ For more information check the java docs for:
 * DB2NGJNDIDataStoreFactory
 * DB2NGDataStoreFactory
 
+Advanced Geotools Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++-------------------+------------------------------------------+
+| Parameter         | Description                              |
++===================+==========================================+
+| "loose bbox"      | Flag controlling loose bbox comparisons, |
+|                   | default is true                          |
++-------------------+------------------------------------------+
+| "use selectivity" | Flag to raise the priority of spatial    |
+|                   | indices for the db2 optimizer,           |
+|                   | default is false                         |
++-------------------+------------------------------------------+
```

Comment:
yay documentation! I am still a bit confused on how it is worded ...

If this is just changing a priority would you do better to use a word here (rather than just a flag).